themes/fast-track-theme/lms/templates/ccx/coach_dashboard_details.html

```diff
@@ -94,16 +94,22 @@ <h2 class="hd hd-2" id="header-ccx-dashboard">${_("COURSE INFORMATION")}</h2>
             </li>
         </ul>
         <section id="schedule" class="idash-section" aria-label="${_('Schedule')}">
```

Comment:
Shouldn't all of these three sections be within one if conidition? will it brake something?